otcextensions/osclient/dns/v2/recordset.py

```diff
@@ -50,16 +50,24 @@ def get_parser(self, prog_name):
             help=_('UUID or name of the zone. Recordsets of all zones '
                    'will be returned if not given.')
         )
+        parser.add_argument(
+            '--zone-type',
```

Comment:
using --zone-type makes no difference while using: openstack dns zone list [--zone-type public|private] <zone-id>. If you use "zone list" with or without parameter --zone-type there are no changes. The output is always the same.